### PR TITLE
add tests to raise coverage back to 94%

### PR DESCRIPTION
**Pull request by eclendenning-va**

### CODE REVIEW

File: vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationToInsuranceCompanyFileTransformer.java [hunk 1]
```diff
@@ -176,11 +175,17 @@ private void addRequiredFields(Set<WriteableFilemanValue> fields) {
...
-    /* TODO https://vajira.max.gov/browse/API-10394 require at least 1 line */
-    String[] arrayLines = new String[] {streetLine1, streetLine2, streetLine3};
     if (isBlank(lines)) {
-      lines = emptyList();
+      throw MissingRequiredField.builder().jsonPath(jsonPathBase + ".line[]").build();
     }
+    if (lines.size() > 3) {
+      throw UnexpectedNumberOfValues.builder()
+          .jsonPath(jsonPathBase + ".line[]")
+          .expectedCount(3)
```
Comment (jhulbert-va) [inline vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationToInsuranceCompanyFileTransformer.java:184]: This isn't true every time right?

Comment (eclendenning-va) [reply vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationToInsuranceCompanyFileTransformer.java:184]: if they are over 3 lines this lets them know to cut it down to at least 3

File: vista-fhir-query/src/test/java/gov/va/api/health/vistafhirquery/service/controller/coverageeligibilityresponse/R4CoverageEligibilityResponseToVistaFileTransformerTest.java [hunk 2]
```diff
@@ -32,6 +33,30 @@ private R4CoverageEligibilityResponseToVistaFileTransformer _transformer() {
         .build();
   }
 
+  @Test
+  void extractFromCodeableConceptOrDie() {
+    assertThatExceptionOfType(MissingRequiredField.class)
+        .isThrownBy(() -> _transformer().extractFromCodeableConceptOrDie("", "", "", "", null));
```
Comment (jhulbert-va) [inline vista-fhir-query/src/test/java/gov/va/api/health/vistafhirquery/service/controller/coverageeligibilityresponse/R4CoverageEligibilityResponseToVistaFileTransformerTest.java:39]: Not sure how I feel about this. I would expect fake values exception for the one you're using to cause the error

File: vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationToInsuranceCompanyFileTransformer.java (hunk 1 above)
Comment (jhulbert-va) [reply vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationToInsuranceCompanyFileTransformer.java:184]: The error will tell them that 3 are expected, not necessarily that 1 or 2 are acceptable. Perhaps this should throw a conditional one with a better message.

File: vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/RequestPayloadExceptions.java [hunk 3]
```diff
@@ -176,15 +176,24 @@ private static UnexpectedNumberOfValues unexpectedNumberOfValues(
         @NonNull String jsonPath,
         String identifyingFieldJsonPath,
         String identifyingFieldValue,
-        int expectedCount,
+        int exactExpectedCount,
+        int atLeastExpectedCount,
+        int atMostExpectedCount,
```
Comment (jhulbert-va) [inline vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/RequestPayloadExceptions.java:181]: minimum and maximum?

File: vista-fhir-query/src/test/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationToInsuranceCompanyFileTransformerTest.java [hunk 4]
```diff
@@ -25,6 +26,89 @@ private R4OrganizationToInsuranceCompanyFileTransformer _transformer() {
         .build();
   }
 
+  @Test
+  void address() {
+    assertThatExceptionOfType(MissingRequiredField.class)
+        .isThrownBy(
+            () ->
+                _transformer().address("", "", "", "", "", "", "", "", Address.builder().build()));
```
Comment (jhulbert-va) [inline vista-fhir-query/src/test/java/gov/va/api/health/vistafhirquery/service/controller/organization/R4OrganizationToInsuranceCompanyFileTransformerTest.java:34]: Same here with the blank values.

**Review by bschofield-va — APPROVED** [review]

**Pull request merged**